_toc.yml

```diff
@@ -582,19 +582,30 @@ entries:
       - file: docs/products/clickhouse/howto
         title: HowTo
         entries:
-          - file: docs/products/clickhouse/howto/connect-to-ch-service
-          - file: docs/products/clickhouse/howto/secure-ch-service
+          - file: docs/products/clickhouse/howto/connect-with-clickhouse-cli
+            title: Connect with CLI
+          - file: docs/products/clickhouse/howto/secure-service
+            title: Secure a service
           - file: docs/products/clickhouse/howto/manage-users-roles
+            title: Manage users and roles
           - file: docs/products/clickhouse/howto/load-dataset
-          - file: docs/products/clickhouse/howto/query-ch-dbs
-          - file: docs/products/clickhouse/howto/manage-dbs-tables
+            title: Load data
+          - file: docs/products/clickhouse/howto/query-databases
+            title: Query databases
+          - file: docs/products/clickhouse/howto/manage-databases-tables
+            title: Manage databases and tables
           - file: docs/products/clickhouse/howto/materialized-views
+            title: Use materialized views
           - file: docs/products/clickhouse/howto/list-integrations
             entries:
               - file: docs/products/clickhouse/howto/connect-to-grafana
+                title: Connect to Grafana®
```

Comment:
In general, we don't use `®` marks in the table-of-contents sidebar, as it adds to the visual clutter in a narrow space, and makes it harder to read.
The text in the sidebar is something we're going to look at properly at some stage, so I'd leave the `®` marks in the `title:` lines out for now (this applies to the next couple of changes below as well)